src/main/scala/com/gu/newsletterlistcleanse/braze/BrazeClient.scala

```diff
@@ -33,7 +33,9 @@ class BrazeClient {
     val results = block
 
     results.foreach {
-      case Right(successResult) => logger.info(s"BrazeClient success: $info $successResult")
+      case Right(successResult) =>
+        logger.info(s"BrazeClient success: $info")
+        logger.debug(s"BrazeClient result: $successResult")
```

Comment:
Where do we set debug mode out of interest?

Reply:
That depends on the logging library. We use Log4j2 here, so in the log4j2.xml